content/en/role-based-access-control.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Role-Based Access Control (RBAC)
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Role-Based Access Control (RBAC) is a method to regulate system and resource access based on individual users' roles within an organization.
+Each job function within an organization has a different role with specific permissions. 
+For instance, a marketing team member may have permission to view the deals sales has in the pipeline (*read* permission) but won't be allowed to edit them (*write* permission). 
+Sales and marketing have different permissions based on their roles, hence role-based access control. 
+
+## Problem it addresses
+
+Individually managing access permissions to many users in an organization of various system resources and data is very complex considering that each user will have different set of resources the user can access. For instance, if there are 100,000 users and 1,000 resources in an organization, an administrator needs to check whether each user has authority and responsibility for each resource 100,000 times.
```

Comment:
```suggestion
Individually managing access permissions of multiple users across various system resources and data can be complex.
After all, each user likely needs access to a different set of resources. 
Let's say an organization has 500 developers (users) and 300 microservices with varying permission levels (read, write, and no access). The administrator must ensure each user has the correct permissions for all 500 resources.  
```
I changed the example to a smaller group. 100,000 is a huge organization. Most people work at much smaller companies. 500 developers is still big but might feel more real. We can also decrease that number to 50 (?)